scripts/README.md

```diff
@@ -51,7 +51,7 @@ Making the requests to the gnosis-interfaces does not cost any gas. However, sig
 
 Here is an example script invocation:
 
-```js
```

Comment:
does this not highlight syntax propoerly or what is the reason for removing it?

Reply:
Ideally all of these commands should be tagged with `sh` to highlight as shell command.

Reply:
Well, it's not js code, so I removed that. I wanted to replace is with something representing the terminal, but then I'd have had to choose between `sh`, `bash`, and `zsh`.
Let's see:

Reply:
JS:
```js
npx truffle exec scripts/complete_liquidity_provision.js --baseTokenId=1 --quoteTokenId=4 --lowestLimit=150 --highestLimit=200 --currentPrice=175 --masterSafe=$MASTER_SAFE --depositBaseToken=0.1 --depositQuoteToken=10 --numBrackets=10 --network=$NETWORK_NAME
```
Bash:
```bash
npx truffle exec scripts/complete_liquidity_provision.js --baseTokenId=1 --quoteTokenId=4 --lowestLimit=150 --highestLimit=200 --currentPrice=175 --masterSafe=$MASTER_SAFE --depositBaseToken=0.1 --depositQuoteToken=10 --numBrackets=10 --network=$NETWORK_NAME
```
Zsh:
```zsh
npx truffle exec scripts/complete_liquidity_provision.js --baseTokenId=1 --quoteTokenId=4 --lowestLimit=150 --highestLimit=200 --currentPrice=175 --masterSafe=$MASTER_SAFE --depositBaseToken=0.1 --depositQuoteToken=10 --numBrackets=10 --network=$NETWORK_NAME
```
Sh:
```sh
npx truffle exec scripts/complete_liquidity_provision.js --baseTokenId=1 --quoteTokenId=4 --lowestLimit=150 --highestLimit=200 --currentPrice=175 --masterSafe=$MASTER_SAFE --depositBaseToken=0.1 --depositQuoteToken=10 --numBrackets=10 --network=$NETWORK_NAME
```